_http_common.d.ts

```diff
@@ -10,13 +11,15 @@ declare var HTTPParser: {
 export interface HTTPParser<ParserType extends number> {
   new (): HTTPParser<ParserType>
 
+  [HTTPParser.kOnHeaders]: (headers: Array<string>, url: string) => void
   [HTTPParser.kOnHeadersComplete]: ParserType extends 0
     ? RequestHeadersCompleteCallback
     : ResponseHeadersCompleteCallback
   [HTTPParser.kOnBody]: (chunk: Buffer) => void
   [HTTPParser.kOnMessageComplete]: () => void
+  maxHeadersPair: number
 
-  initialize(type: ParserType, asyncResource: object): void
+  initialize(type: ParserType, asyncResource: object, maxHeaderSize?: number, lenient?: number, headersTimeout?: number): void
```

Comment:
Could you please point me to Node.js source where you saw this additional arguments? I'm struggling to find them...

Reply:
It didn't have any particular meaning. I just aligned with Node.js types.